### PR TITLE
Reduce logging from start script

### PR DESCRIPTION
**Pull request by heppu**

### CODE REVIEW

**Review by rofafor — APPROVED** [review]

File: keydb/templates/secret-utils.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ type: Opaque
 stringData:
   server.sh: |
     #!/bin/sh
-    set -euxo
+    set -euo pipefail
```
Comment (rofafor) [inline keydb/templates/secret-utils.yaml:11]: We might want to log the generated command-line arguments without actual REDIS_PASSWORD shown

Comment (heppu) [reply keydb/templates/secret-utils.yaml:11]: Any suggestions/preferred ways how to do that?

Comment (heppu) [reply keydb/templates/secret-utils.yaml:11]: Now it looks like this:
```sh
$ kubectl logs -n henri pod/keydb-0

Defaulted container "keydb" out of: keydb, redis-exporter
keydb-server /etc/keydb/redis.conf --active-replica yes --multi-master yes --appendonly no --bind 0.0.0.0 --port 6379 --protected-mode no --server-threads 2 --masterauth *** --requirepass *** --replicaof keydb-1.keydb-headless 6379 --replicaof keydb-2.keydb-headless 6379
1:1:C 29 Jun 2023 09:10:43.541 * Notice: "active-replica yes" implies "replica-read-only no"
```

Comment (heppu) [reply keydb/templates/secret-utils.yaml:11]: And without REDIS_PASSWORD
```sh
$ kubectl logs -n henri pod/keydb-0

Defaulted container "keydb" out of: keydb, redis-exporter
keydb-server /etc/keydb/redis.conf --active-replica yes --multi-master yes --appendonly no --bind 0.0.0.0 --port 6379 --protected-mode no --server-threads 2 --replicaof keydb-1.keydb-headless 6379 --replicaof keydb-2.keydb-headless 6379
1:1:C 29 Jun 2023 09:28:36.138 * Notice: "active-replica yes" implies "replica-read-only no"
```

File: keydb/templates/secret-utils.yaml [hunk 2]
```diff
@@ -46,4 +46,10 @@ stringData:
         {{- end }}
         {{- end }}
         {{- end }}
-        $(echo "${replicas}" | xargs)
+        $(echo "${replicas}" | xargs))
+    {{- if .Values.existingSecret }}
```
Comment (rofafor) [inline keydb/templates/secret-utils.yaml]: This isn't enough as there are also `.Values.password` affecting to `REDIS_PASSWORD`.  Just check whether `REDIS_PASSWORD` is non-empty.

**Review by rofafor — CHANGES_REQUESTED** [review]

File: keydb/templates/secret-utils.yaml (hunk 2 above)
Comment (heppu) [reply keydb/templates/secret-utils.yaml]: Fixed based on this answer: https://stackoverflow.com/a/13864829

**Review by rofafor — APPROVED** [review]

**Pull request merged**